### PR TITLE
Open8 Bid Adapter: Add `advertiserDomains` support to comply with Prebid.js 5.0

### PR DESCRIPTION
**Pull request by hisui**
## Type of change

- [x] Other

## Description of change

- test parameters for validating bids

```js
{
    bidder: 'open8',
    params: {
        slotKey: '2ae5a533'
    }
}
```

- contact email of the adapter’s maintainer: tdd-adtech@open8.com
- [x] official adapter submission

### CODE REVIEW

File: modules/open8BidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,188 @@
+import { Renderer } from '../src/Renderer.js';
+import {ajax} from '../src/ajax.js';
+import * as utils from '../src/utils.js';
```
Comment (patmmccann) [inline modules/open8BidAdapter.js:3]: Please only import what you need

File: modules/open8BidAdapter.js [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+import { VIDEO, BANNER } from '../src/mediaTypes.js';
+
+const BIDDER_CODE = 'open8';
+const URL = '//as.vt.open8.com/v1/control/prebid';
```
Comment (patmmccann) [inline modules/open8BidAdapter.js:8]: We no longer support protocol relative, use secure

**Review by patmmccann — CHANGES_REQUESTED** [review]
Couple change requests

**Comment by hisui** [comment]
@patmmccann Thank you for the review. I've fixed them as you indicated.

**Review by ChrisHuie — APPROVED** [review]

**Pull request merged**